controllers/articles.controller.js

```diff
@@ -63,4 +63,25 @@ exports.patchArticle = (req, res, next) => {
       res.status(200).send( updatedArticle );
     }).catch(next)
   
-  };
\ No newline at end of file
+  };
+
+
+  
+  exports.deleteComment =  (req, res, next) =>
+    {
+      const comment_id = req.params.comment_id
+      
+     
+        removeComment(comment_id) 
+        .then((deletedata) => {
+          if(deletedata.rowCount === 0){
+           next({status: 404, msg:"Comment does not exist"})
+          }else res.status(204).send()
```

Comment:
I'm not completely against this logic in the controller, but I do feel the checking of the comment existing and the rejecting of the promise should be in the model. But this does work and if you like it the way it is, that is fine